appengine/standard/ndb/schema_update/add_entities_handler.py

```diff
@@ -0,0 +1,32 @@
+import os
+import time
+
+import jinja2
+import webapp2
+
+import update_schema
+
+
+JINJA_ENVIRONMENT = jinja2.Environment(
+    loader=jinja2.FileSystemLoader(os.path.dirname(__file__)),
+    extensions=['jinja2.ext.autoescape'],
+    autoescape=True)
+
+class AddEntitiesHandler(webapp2.RequestHandler):
+    def post(self):
+        author1 = self.request.get('author1', 'bob')
```

Comment:
would it just be easier not to use request data but just insert some canned entities?

```python
ndb.put_multi([
     models_v1.Picture(author='alice@example.com', name='Alice'),
     models_v1.Picture(author='bob@example.com', name='Bob')
])
```